src/views/admin-results-listing.phtml

```diff
@@ -69,17 +70,28 @@
             </table>
         </div>
 
```

Comment:
Why not use the paginator directly? as in `$paginator->getCurrentPage()` instead of `if ($currentPage === 1)` so on and so forth

Reply:
I thought it's easier to read and less method calls

Reply:
In my opinion readability is not improved, on the contrary. 
Coming from the controller I know that I have paginator object and what values it has. When I reach the template I need to find were this local variables are set and keep track of them while I browse the template. Use paginator object directly. It's a micro performance improvement as well as you do not additional memory for the same values